src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -370,15 +409,18 @@ public async Task SynchronizeWithServer_UnsupportedServerVersion_UIContextIsClea
             const uint cookie = 999;
             var sonarQubeServer = CreateSonarService(isConnected: true, serverType: ServerType.SonarQube, versionString);
             var monitorMock = CreateMonitorSelectionMock(cookie);
+            var logger = new TestLogger();
 
             var testSubject = CreateTestSubject(
+                bindingConfig: BindingConfig_Connected,
                 sonarService: sonarQubeServer.Object,
-                mode: SonarLintMode.Connected,
-                vsMonitor: monitorMock.Object);
+                vsMonitor: monitorMock.Object,
+                logger: logger);
 
             await testSubject.SynchronizeWithServer();
 
             CheckUIContextUpdated(monitorMock, cookie, 0);
+            logger.AssertPartialOutputStringExists(versionString);
```

Comment:
Additional check: the unsupported server version should be logged as part of a message to the user.